tekton/pipeline_run.go

```diff
@@ -50,7 +51,7 @@ const (
 	PipelineTypeRelease = "release"
 
 	// pipelinesAsCodeMetadataPrefix is the prefix for pipelines-as-code metadata
-	pipelinesAsCodeMetadataPrefix = "pipelinesascode.tekton.dev"
+	pipelinesAsCodeMetadataPrefix = integrationServiceGitopsPkg.PipelinesAsCodePrefix
```

Comment:
I think it would be better to not define this constant here and instead use `integrationServiceGitopsPkg.PipelineAsCodePrefix` everywhere `pipelinesAsCodeMetadataPrefix` is currently used

Reply:
That works great for me. Thanks.